stdlib/gc.mli

```diff
@@ -145,17 +145,46 @@ type control =
     mutable allocation_policy : int;
     [@ocaml.deprecated_mutable
          "Use {(Gc.get()) with Gc.allocation_policy = ...}"]
-    (** The policy used for allocating in the heap.  Possible
-        values are 0 to 2.  0 is the original next-fit policy,
-        usually fast, but can result in fragmentation.  1 is the
-        first-fit policy, which can be slower in some cases but
-        can be better for programs with fragmentation problems.
-        2 is the best-fit allocation policy, the best for avoiding
-        fragmentation.
-        Note that changing the allocation policy at run-time forces
+    (** The policy used for allocating in the major heap.
+        Possible values are 0, 1 and 2.
+
+        - 0 is the next-fit policy, which is usually fast but can
+          result in fragmentation.
+
+        - 1 is the first-fit policy, which avoids fragmentation but
+          is sensibly slower in some real-world use-cases (it is only
+          useful on programs exhibiting fragmentation issues with
+          next-fit)
+
+        - 2 is the best-fit policy, which is empirically better than
+          next-fit and first-fit: in our experiments it is faster and
+          uses less memory than both.
+
+        The current default is next-fit, as the better best-fit policy
+        is new and not yet widely tested. We expect best-fit to become
+        the default in the future.
+
+        On one example that was known to be bad for next-fit and first-fit,
+        next-fit takes 28s using 855Mio of memory,
+        first-fit takes 47s using 566Mio of memory,
+        best-fit takes 27s using 545Mio of memory.
+
+        Note: the difference in fragmentation behavior means that
+        different policies will have different proportion of "wasted
+        space" for a given program: less compaction means a smaller
+        heap, so (for the same total memory consumption) more wasted
```

Comment:
tuned ⇒ is tuned

    less compaction means a smaller heap

This is confusing: I am not sure to follow how less compaction imply a smaller heap?

Reply:
"less fragmentation", sorry, will fix